### PR TITLE
test: adds missing adapter tests

### PR DESCRIPTION
**Pull request by reecejohnson**
Adds missing tests for recently introduced adapters.

### CODE REVIEW

File: packages/api-bindings/src/mocks.ts [hunk 1]
```diff
@@ -1,2 +1,3 @@
 export * from './graphql/__helpers__';
 export * from './apollo/__helpers__/mocks';
+export { createProfileTypePolicy } from './apollo/createProfileTypePolicy';
```
Comment (reecejohnson) [inline packages/api-bindings/src/mocks.ts:3]: We need to access this in `FollowProfilesResponder.spec.ts` as part of the Apollo client setup.

Although not strictly a mock, we shouldn't be exporting as part of the main package

File: packages/react/src/transactions/adapters/responders/__tests__/FollowProfilesResponder.spec.ts [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+import { FollowProfilesResponder } from '../FollowProfilesResponder';
+
+function setupTestScenario({ existingProfile }: { existingProfile: ProfileFieldsFragment }) {
+  const apolloCache = new InMemoryCache({
+    addTypename: true,
+    typePolicies: {
+      Profile: createProfileTypePolicy(),
+    },
+  });
```
Comment (cesarenaldi) [inline packages/react/src/transactions/adapters/responders/__tests__/FollowProfilesResponder.spec.ts:20]: I created the `createMockApolloCache` exactly for this purpose. It gives you an `InMemory` cache with all the type policies and possible types wired in.

File: packages/api-bindings/src/mocks.ts (hunk 1 above)
Comment (cesarenaldi) [reply packages/api-bindings/src/mocks.ts:3]: We can avoid this by using the `createMockApolloCache` helper. See my other comment.

**Review by desfero — APPROVED** [review]
Some of the tests would fail when we merge #71, right @reecejohnson ?

**Pull request merged**